This is to disable SIMD optimization until SAFARI has good support for it.


diff --git a/numpy/distutils/command/build.py b/numpy/distutils/command/build.py
index a4fda537d..11fffe282 100644
--- a/numpy/distutils/command/build.py
+++ b/numpy/distutils/command/build.py
@@ -37,7 +37,7 @@ def initialize_options(self):
         self.warn_error = False
         self.cpu_baseline = "min"
         self.cpu_dispatch = "max -xop -fma4" # drop AMD legacy features by default
-        self.disable_optimization = False
+        self.disable_optimization = True
         """
         the '_simd' module is a very large. Adding more dispatched features
         will increase binary size and compile time. By default we minimize
